src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { NavLink, Outlet } from "react-router-dom";
+import { logo } from "../../assets";
+import "./home.css";
+
+function Home() {
+  return (
+    <div className="bg-[#11161b] text-gray-50 h-screen w-screen flex flex-col">
+      <nav className="navbar bg-base-100 px-4 py-2">
+        <div className="flex-none">
+          <img src={logo} alt="logo" className="h-10" />
+        </div>
+        <div className="flex-1">
+          <button className="ff-nickainley text-2xl mx-4">The Social</button>
+        </div>
+        <div className="dropdown dropdown-end">
+          <label tabIndex="0" className="btn m-1 border-0">
+            <div className="indicator text-2xl">
+              <span className="indicator-item badge rounded-full ">4</span>
+              <div className="grid w-10 h-10 bg-base-300 place-items-center">
+                <i className="fa-solid fa-bell"></i>
+              </div>
+            </div>
+          </label>
+          <ul
+            tabIndex="0"
+            className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
+            <li className="bg-[#454d50] hover:bg-[#2c3133]">
+              <p>Notification 1</p>
+            </li>
+            <li className="bg-[#454d50] hover:bg-[#2c3133]">
+              <p>Notification 2</p>
+            </li>
+          </ul>
+        </div>
+        <div className="dropdown dropdown-end  ">
+          <label tabIndex="0" className="btn m-1 bg-[#424F55] outline-none border-0">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              className="inline-block w-5 h-5 stroke-current">
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth="2"
+                d="M5 12h.01M12 12h.01M19 12h.01M6 12a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0z"></path>
+            </svg>
```

Comment:
**suggestion**: in place of using SVGs directly in the code, try importing it

Reply:
this is taken from library (Daisy UI)